spec/support/messaging/en.yml

```diff
@@ -134,7 +134,7 @@ en:
           'yes': 'Yes'
           'no': 'No'
       disagree_claimant_notice_reason:
-        label: Tell us the details you believe to be correct.
```

Comment:
I have just noticed that you have not modified the welsh version also.  When we run this test suite in CI, we also run it in welsh (it adds TEST_LOCALE=cy to the command line)

This is more than likely why the test are failing ?